frontend/src/types.ts

```diff
@@ -629,7 +629,7 @@ export interface FilterType {
     actions?: Record<string, any>[]
     breakdown_type?: BreakdownType | null
     breakdown?: string | number | number[] | null
-    breakdown_value?: string
+    breakdown_value?: string | number
```

Comment:
Does the frontend send this to BE in some cases? If so, when?

Reply:
when you retrieve persons now you use `breakdown_value`. Functionally, `breakdown` will be a display value and `breakdown_value` will be the underlying id if there's a difference. so for property breakdown both breakdown and breakdown_value will be the same but for cohort breakdowns, the returned `breakdown` will be the name of the cohort and `breakdown_value` will be the id of the cohort

this is all fairly confusing because `breakdown` sort of purpose swaps when being passed into the API call and then when later used on the return